NitroxServer/Serialization/ServerConfig.cs

```diff
@@ -43,6 +43,9 @@ public int SaveInterval
             }
         }
 
+        [PropertyDescription("Command to run following a successful word save (e.g. world backup .exe, .bat, or PowerShell script)")]
+        public string PostSaveCommandPath { get; set; } = string.Empty;
```

Comment:
Could override this setter to trim the given value so that other parts of code don't need to do it.